semantic-python/semantic-python.cabal

```diff
@@ -45,7 +46,10 @@ common haskell
 
 library
   import: haskell
-  exposed-modules: Language.Python.Core
+  exposed-modules:
+    Language.Python
+    Language.Python.Core
+    Language.Python.Tags
```

Comment:
I’ve taken the opportunity to move `Language.Python` & `Language.Python.Tags` from `semantic-tags` into `semantic-python`.